Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1636,7 +1631,21 @@ extension BrowserViewController: TabManagerDelegate {
             ReaderModeHandlers.readerModeCache = readerModeCache
 
             scrollController.browser = selected
+
+            // before we add a new webView, ensure that there aren't too many subviews already in the webViewContainer
+            if webViewContainer.subviews.count > 20,
+                let bottomWebView = ( webViewContainer.subviews.find {
```

Comment:
Yup you're right - I should probably drink my :coffee: before reviewing :disappointed: